src/postgres/tree/ClientConfigFactory.ts

```diff
@@ -22,14 +23,20 @@ export class PostgresClientConfigFactory {
         clientConfig: ClientConfig
     }> {
         const parsedConnectionString = await treeItem.getFullConnectionString();
-        const azureUserSession = await getAzureAdUserSession();
```

Comment:
This no longer works for most of the users because the authentication has been moved out of Account extension.